lib/common/strings.c

```diff
@@ -238,15 +238,6 @@ pcmk__scan_double(const char *text, double *result, const char *default_text,
             crm_debug("Floating-point value parsed from '%s' would %sflow "
                       "(using %g instead)", text, over_under, *result);
 
-        } else if (errno != 0) {
-            rc = errno;
-            // strtod() set *result = 0 on parse failure
-            *result = PCMK__PARSE_DBL_DEFAULT;
-
-            crm_debug("Could not parse floating-point value from '%s' (using "
-                      "%.1f instead): %s", text, PCMK__PARSE_DBL_DEFAULT,
-                      pcmk_rc_str(rc));
-
```

Comment:
errno's are implementation-dependent, so I'd rather keep future-proofing code in there in case some weird compiler on some weird architecture ever does it differently

Reply:
Dropped this patch.

Reply:
If you just want to hit the numbers you could mock the system call :)

Reply:
Hah, yes, I guess I could do that.  Honestly I'm fine leaving it as-is.  This is good enough to make the row in the coverage report green.